src/app/loadout-builder/process-worker/tsconfig.json

```diff
@@ -19,5 +19,6 @@
       "app/*": ["../../*"],
       "data/*": ["../../../data/*"]
     }
-  }
+  },
+  "include": ["./*", "../../../global.d.ts"]
```

Comment:
My vs-code doesn't pick up the global types for the process file. This fixes it but I am not sure if we actually want it. I am hoping you know better than I do.